aquadoggo/src/network/service.rs

```diff
@@ -271,21 +263,6 @@ pub async fn connect_to_relay(
                 info!("Registered on rendezvous in namespace \"{namespace}\"");
                 rendezvous_registered = true;
             }
-            SwarmEvent::ConnectionClosed { .. } => {
```

Comment:
`swarm.listen_on(circuit_address)` no longer sometimes fails so we don't need this retry loop anymore.